python/pyhail/keytable.py

```diff
@@ -50,7 +51,7 @@ def key_names(self):
         :rtype: list of str
         """
         try:
-            return self.jkt.keyNames()
+            return list(self.jkt.keyNames())
```

Comment:
I thought it would be better to coerce the result to a python type. This is the difference in the python shell.
```
>>> kt.field_names()
JavaObject id=o29
>>> list(kt.field_names())
[u'Sample', u'Status', u'qPhen']
```